tools/clang/tools/dxcompiler/dxcshadersourceinfo.h

```diff
@@ -56,7 +56,7 @@ struct SourceInfoWriter {
   Buffer m_Buffer;
 
   const hlsl::DxilSourceInfo *GetPart() const;
-  void Write(llvm::StringRef targetProfile, llvm::StringRef entryPoint, clang::CodeGenOptions &cgOpts, clang::SourceManager &srcMgr);
+  void Write(clang::CodeGenOptions &cgOpts, llvm::ArrayRef<hlsl::options::ArgPair> ArgPairs, clang::SourceManager &srcMgr);
```

Comment:
Params to this function look like they have inconsistent naming conventions to me:
* `cgOpts`
* `ArgPairs`
* `srcMgr`

Reply:
Fixed it.